python/thunder/rdds/keys.py

```diff
@@ -60,67 +60,67 @@ def _indtosub_converter(dims, order='F', onebased=True):
     Series.indtosub
     """
 
-    _check_order(order)
+    _checkOrder(order)
 
-    def indtosub_inline_onebased(k, dimprod):
-        return tuple(map(lambda (x, y): int(mod(ceil(float(k)/y) - 1, x) + 1), dimprod))
+    def indToSub_InlineOneBased(k, dimProd_):
+        return tuple(map(lambda (x, y): int(mod(ceil(float(k)/y) - 1, x) + 1), dimProd_))
 
-    def indtosub_inline_zerobased(k, dimprod):
-        return tuple(map(lambda (x, y): int(mod(ceil(float(k+1)/y) - 1, x)), dimprod))
+    def indToSub_InlineZeroBased(k, dimProd_):
+        return tuple(map(lambda (x, y): int(mod(ceil(float(k+1)/y) - 1, x)), dimProd_))
 
-    inline_fcn = indtosub_inline_onebased if onebased else indtosub_inline_zerobased
+    inlineFcn = indToSub_InlineOneBased if isOneBased else indToSub_InlineZeroBased
 
     if size(dims) > 1:
         if order == 'F':
-            dimprod = zip(dims, append(1, cumprod(dims)[0:-1]))
+            dimProd = zip(dims, append(1, cumprod(dims)[0:-1]))
         else:
-            dimprod = zip(dims, append(1, cumprod(dims[::-1])[0:-1])[::-1])
-        converter = lambda k: inline_fcn(k, dimprod)
+            dimProd = zip(dims, append(1, cumprod(dims[::-1])[0:-1])[::-1])
+        converter = lambda k: inlineFcn(k, dimProd)
     else:
         converter = lambda k: (k,)
 
     return converter
 
 
-def _subtoind_converter(dims, order='F', onebased=True):
+def _subToIndConverter(dims, order='F', isOneBased=True):
     """Converter for changing subscript indexing to linear indexing
 
     See also
     --------
     Series.subtoind
     """
-    _check_order(order)
+    _checkOrder(order)
 
-    def onebased_prod(x_y):
-        x, y = x_y
+    def onebasedProd(xy):
+        x, y = xy
         return (x - 1) * y
 
-    def zerobased_prod(x_y):
-        x, y = x_y
+    def zerobasedProd(xy):
+        x, y = xy
         return x * y
 
-    def subtoind_inline_colmajor(k, dimprod, p_func):
-        return sum(map(p_func, zip(k[1:], dimprod))) + k[0]
+    def subToInd_InlineColmajor(k, dimProd_, prodFunc):
```

Comment:
Hobgoblin of little minds, @freeman-lab. ;)

Yeah, here (and to some degree in the other remaining underscore) I thought that the underscore was useful in separating components of the name. subToIndInlineColmajor - are "subs" being converted to "IndInlineColmajors"? This version I think keeps it more clear that it's the "InlineColmajor" flavor of "subToInd"